Telegram/SourceFiles/kotato/settings_menu.cpp

```diff
@@ -313,6 +313,7 @@ void SetupKotatoMessages(not_null<Ui::VerticalLayout*> container) {
 	AddSkip(container);
 
 	SettingsMenuSwitch(ktg_settings_adaptive_bubbles, AdaptiveBubbles);
+	SettingsMenuSwitch(ktg_settings_monospace_large_bubbles, MonospaceLargeBubbles);
```

Comment:
Since the option has effect only when adaptive bubbles are disabled, it would be nice to un-macro both, and make `Ui::SlideWrap`, so it will be shown only if adaptive bubbles are disabled.

Reply:
Examples of `Ui::SlideWrap` usage:

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L557-L561

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L566-L571

https://github.com/kotatogram/kotatogram-desktop/blob/f9ad9ce03bba67845e65e14295dbe8a098afcadb/Telegram/SourceFiles/settings/settings_notifications.cpp#L666-L668

Reply:
Actually, I'll do it later by myself. Thanks anyway.